sycl/include/sycl/detail/pi.h

```diff
@@ -94,9 +94,11 @@
 // info query.
 // 12.32 Removed backwards compatibility of piextQueueCreateWithNativeHandle and
 // piextQueueGetNativeHandle
+// 13.33 Added piextEnablePeerAccess, piextDisablePeerAccess,
+// piextPeerAccessGetInfo, and pi_peer_attr enum.
```

Comment:
We need to stop voluntarily changing PI API, without prior discussion/agreement with Unified Runtime.
@kbenzie, has this been brought up before? are you taking this into UR API?